### ISSUE TITLE
Test gt missing and val param problem

### ISSUE THREAD

**Issue by RogerJaeger** [EGO4D/ego-exo4d-relation#2]
Thank you for your work. However, I encountered some issues during reproduction and hope you can help me address them.
First, I didn't find the ground truth for the test set in the annotations files. Specifically, the test set defined in split.json does not seem to have corresponding ground truth annotations. I only obtained relations_train.json and relations_val.json. Is this intentional, or am I missing something?
Second, in the file correspondence/SegSwap/train/Main.py at line 207, trainLoader is used instead of valLoader. Is this a typo?

![Image](https://github.com/user-attachments/assets/7a5e44d1-adab-48bd-8b20-139ed19bf30e)

**Comment by sanjayharesh** [comment]
Hi @RogerJaeger ,

Sorry for the late response. 
For first, no you are not missing anything. Unfortunately the test annotations have not been released yet. I have reached out to the team handling the dataset to see if we can release them soon. I will let you know when I get a response from them. In the meanwhile to unblock you, attaching the results on val set for XSegTx baseline on both ego->exo and exo->ego settings.

![Image](https://github.com/user-attachments/assets/59591b0e-4e3e-4871-9087-0131a7f0b727)

For second, thanks for catching that -- fixed it with the latest update along with a bit of cleanup. For what it's worth, evals on val set were run outside this script so it should not affect the results.

**Comment by RogerJaeger** [comment]
Hi @sanjayharesh ,

Thank you for your reply. 
Hope to receive the response about the test dataset from you.

**Comment by sanjayharesh** [comment]
The test annotations have been released now. Please re-download the relations annotations using the following command,
`egoexo --parts annotations --release v2 -o ./ego4d_data/ --benchmarks relations`.

**Comment by lmur98** [comment]
Hi, could you share the full Table with all the results in the validation set?
